dataGenerator.js

```diff
@@ -23,149 +23,931 @@ var db = require('./server/db.js');
 var users = require('./server/users.js');
 var questions = require('./server/questions.js');
 var common = require('./server/common.js');
+var contents = require('./contents.js');
 
 // variables to control the genereated data
+var numberOfEachQuestion = [2, 2, 2, 2, 2, 2];
```

Comment:
a dictionary would be nice here. really hard to tell which number is for which question

Reply:
done